### PR TITLE
LIVE-33/34: Retrieve list of participants and display it

### PR DESCRIPTION
**Pull request by LVala**
Added React component ParticipantsList which is a dynamic list of participants (currently with mocked photo and options button, nothing styled). Data about participants is retrieved from Presence via client side phoenix library.

### CODE REVIEW

**Comment by LVala** [comment]
Maybe it would be beneficial to write some tests for the channel/presence.

**Comment by Rados13** [comment]
> Maybe it would be beneficial to write some tests for the channel/presence.

Yup I agree with that.

File: assets/js/components/ParticipantsList.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  IconButton,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from "@chakra-ui/react";
+
+const ParticipantMenu = (props) => {
+  return (
+    <Menu>
+      <MenuButton as={IconButton} aroa-label="Options" icon={"Menu Icon"} variant="outline" />
```
Comment (Rados13) [inline assets/js/components/ParticipantsList.tsx:22]: I would change the color of this button to make it easier to find.

File: assets/js/components/ParticipantsList.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+const Participant = ({ name }) => {
+  return (
+    <Flex width="15vw" alignItems="center" gap="1">
+      <Avatar name="Dan Abrahmov" src="https://bit.ly/dan-abramov" />
+      <Box p="2">
+        <Heading size="md">{name}</Heading>
+      </Box>
+      <Spacer />
+      <ParticipantMenu />
```
Comment (Rados13) [inline assets/js/components/ParticipantsList.tsx:42]: I would also think about that moderator and viewer will have different options available (I am not sure if viewer will have any options). So I would add some logic about changing options based on type of the user.

**Review by Rados13 — CHANGES_REQUESTED** [review]
You could also add mentioned tests for Phoenix.Presence.

File: assets/js/components/ParticipantsList.tsx (hunk 1 above)
Comment (LVala) [reply assets/js/components/ParticipantsList.tsx]: I changed the color but everything style-wise is temporary anyway

File: assets/js/components/ParticipantsList.tsx (hunk 2 above)
Comment (LVala) [reply assets/js/components/ParticipantsList.tsx]: Added prop to the component telling if user is a moderator.

**Pull request merged**